topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: tutorial_hands_on
+topic_name: epigenetics
+tutorial_name: hicexplorer
+---
+
+# Introduction
+{:.no_toc}
+
+In this HiCExplorer tutorial we will generate and plot a Hi-C contact matrix.
+For this the following steps are necessary to be performed:
+1. Map the Hi-C reads to the reference genome
+2. Creation of a Hi-C matrix
+3. Plotting the Hi-C matrix
+4. Correction of Hi-C matrix
+5. TAD Calling
+
+After a corrected Hi-C matrix is created other tools can be used to visualize it, call TADS or compare it with other matrices.
+This tutorial is based on the [following work](http://hicexplorer.readthedocs.io/en/latest/content/example_usage.html).
+
+We will be using a Hi-C dataset from [Marks et. al. 2015](http://www.genomebiology.com/2015/16/1/149), on mouse ESCs.
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Data upload
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history
+> 2. Import the [SRR1956527_1.fastq](ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/007/SRR1956527/SRR1956527_1.fastq.gz), [SRR1956527_2.fastq](ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/007/SRR1956527/SRR1956527_2.fastq.gz), [SRR1956528_1](ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/008/SRR1956528/SRR1956528_1.fastq.gz), [SRR1956528_2](ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/008/SRR1956528/SRR1956528_2.fastq.gz), [SRR1956529_1](ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/009/SRR1956529/SRR1956529_1.fastq.gz) and [SRR1956529_2](ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/009/SRR1956529/SRR1956529_2.fastq.gz).
+>
+>    > ### {% icon tip %} Tip: Importing data via links
+>    >
+>    > * Copy the link location
+>    > * Open the Galaxy Upload Manager
+>    > * Select **Paste/Fetch Data**
+>    > Below are the links to the read files that can be copied and pasted in the upload manager.
+>    >
+>    > ``` ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/007/SRR1956527/SRR1956527_1.fastq.gz
+>    > ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/007/SRR1956527/SRR1956527_2.fastq.gz
+>    > ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/008/SRR1956528/SRR1956528_1.fastq.gz
+>    > ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/008/SRR1956528/SRR1956528_2.fastq.gz
+>    > ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/009/SRR1956529/SRR1956529_1.fastq.gz
+>    > ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/009/SRR1956529/SRR1956529_2.fastq.gz
+>    >```
+>    >

```

Comment:
it is now:
.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1176070.svg
   :target: https://doi.org/10.5281/zenodo.1176070

Please exchange the data.